### PR TITLE
Findbugs fixes

### PR DESCRIPTION
**Pull request by kensipe**
findbugs added to the project and all findbug issues fixed.

### CODE REVIEW

**Comment by kensipe** [comment]
@mohitsoni @smarella or @adam-mesos  any help with a review on this?  I would like to move to adding pmd and checkstyle support.

**Comment by kensipe** [comment]
@mohitsoni + team.  I've rebased the recent changes and removed the conflict.  This is ready to go.

**Comment by mohitsoni** [comment]
Also, can you please squash all changes into 1 commit ?

File: Vagrantfile [hunk 1]
```diff
@@ -49,6 +49,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = "ubuntu/trusty64"
   config.vm.box_url = "https://vagrantcloud.com/ubuntu/boxes/trusty64"
 
+  # Consistent IP for demos and tutorials (10.141.141.10 is mesosphere playa address)
+  config.vm.network :private_network, ip: "10.141.141.20"
+
+  # Consistent IP for demos and tutorials (10.141.141.10 is mesosphere playa address)
+  config.vm.network :private_network, ip: "10.141.141.20"
```
Comment (mohitsoni) [inline Vagrantfile]: Looks like a duplicate. Also, I think you sent this one as part of #27

**Comment by kensipe** [comment]
yeah... I'm an idiot.  I rebased on the squash to phase1 and not on it's branch.  let me fix it.

**Comment by kensipe** [comment]
@mohitsoni allll... righty then!

**Pull request merged**